q2_qemistree/plugin_setup.py

```diff
@@ -153,7 +153,7 @@
     inputs={'csi_results': List[CSIFolder],
             'feature_tables': List[FeatureTable[Frequency]],
             'ms2_matches': List[FeatureData[Molecules]]},
-    parameters={'qc_properties': Bool},
+    parameters={'qc_properties': Bool, 'fingerprint_cluster': Str},
```

Comment:
We should make this parameter a `Choice` type and let users select from one of `euclidean` or `jaccard`. Although if we allow for the Jaccard's metric then maybe we should also add a step where we binarize the values.